features/guidance/collapse-ferry.feature

```diff
@@ -0,0 +1,90 @@
+@routing  @guidance @collapsing
+Feature: Collapse
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 20 meters
+
+    Scenario: Collapse Steps While On Ferry
```

Comment:
could you maybe directly add the `ascii-art` to make the tests more readable to begin with? (here and on the other tests)